addons/l10n_pl/tests/test_move.py

```diff
@@ -0,0 +1,58 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from freezegun import freeze_time
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.tests import tagged
+from odoo import fields
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestAccountPL(AccountTestInvoicingCommon):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref='pl'):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+
+        with freeze_time('2024-07-10'):
+            cls.today = fields.Date.today().strftime('%Y-%m-%d')
+
+        cls.dollar_currency = cls.env['res.currency'].search([('name', '=', 'USD')])
+        cls.PL_currency = cls.env['res.currency'].search([('name', '=', 'PLN')])
+        cls.invoice_a = cls.env['account.move'].create({
+            'move_type': 'out_invoice',
+            'partner_id': cls.partner_a.id,
+            'date': cls.today,
+            'invoice_date': cls.today,
+            'currency_id': cls.dollar_currency.id,
+            'company_id': cls.env.company.id,
```

Comment:
```suggestion
            'date': '2024-07-10',
            'invoice_date': '2024-07-10',
            'currency_id': cls.currency_usd.id,
            'company_id': cls.company_data['company'].id,
```

in this case, I think it's better to use a string `'2024-07-10'` for the dates. 
I think it's more readable, and also it doesn't require the freezegun

Reply:
Done and did the same recommendations in CZ PR ^-^